### PR TITLE
Update node service identifier flags

### PR DESCRIPTION
**Pull request by Coding-Enthusiast**
For reference: https://github.com/bitcoin/bitcoin/blob/fa2510d5c1cdf9c2cd5cc9887302ced4378c7202/src/protocol.h#L246-L279

### CODE REVIEW

**Comment by Coding-Enthusiast** [comment]
I kept the values in hex representation to keep it compatible with the docs but isn't it better to change them into something more meaningful like `1 << 10` instead of `0x0400`?

**Comment by harding** [comment]
@Coding-Enthusiast I prefer them in hex because I often use this documentation when inspecting data in, e.g., Wireshark.  That said, I have no objects to adding an extra column to the table with a different representation, or using one representation by default and using a title tag for the other, .e.g. `<span title="1 &lt;&lt; 10">0x0400</span>`

File: _data/devdocs/en/references/p2p_networking.md [hunk 1]
```diff
@@ -1432,6 +1432,11 @@ The following service identifiers have been assigned.
 |-------|--------------|---------------
 | 0x00  | *Unnamed*    | This node is not a full node.  It may not be able to provide any data except for the transactions it originates.
 | 0x01  | NODE_NETWORK | This is a full node and can be asked for full blocks.  It should implement all protocol features available in its self-reported protocol version.
+| 0x02  | NODE_GETUTXO | This is a full node capable of responding to the getutxo protocol request. See BIP-64 for details on how this is implemented.
+| 0x04  | NODE_BLOOM | This is a full node capable and willing to handle bloom-filtered connections.
+| 0x08  | NODE_WITNESS | This is a full node that can be asked for blocks and transactions including witness data.
+| 0x10  | NODE_XTHIN | This is a full node that supports Xtreme Thinblocks.
```
Comment (harding) [inline _data/devdocs/en/references/p2p_networking.md]: Should probably mention that this is not supported by any currently-maintained Bitcoin full node.  The Bitcoin Unlimited website [says](https://www.bitcoinunlimited.info/download) the last version for the BTC network was in 2017.

File: _data/devdocs/en/references/p2p_networking.md [hunk 2]
```diff
@@ -1432,6 +1432,11 @@ The following service identifiers have been assigned.
 |-------|--------------|---------------
 | 0x00  | *Unnamed*    | This node is not a full node.  It may not be able to provide any data except for the transactions it originates.
 | 0x01  | NODE_NETWORK | This is a full node and can be asked for full blocks.  It should implement all protocol features available in its self-reported protocol version.
+| 0x02  | NODE_GETUTXO | This is a full node capable of responding to the getutxo protocol request. See BIP-64 for details on how this is implemented.
```
[the remark this answers is not in the document]
Comment (Coding-Enthusiast) [reply _data/devdocs/en/references/p2p_networking.md]: Improved the PR with your suggestions, now I'm trying to figure out how to add a new commit to change the references file to this PR!

File: _data/devdocs/en/references/p2p_networking.md [hunk 3]
```diff
@@ -1432,6 +1432,11 @@ The following service identifiers have been assigned.
...
+| 0x04  | NODE_BLOOM | This is a full node capable and willing to handle bloom-filtered connections.
```
[the remark this answers is not in the document]
Comment (Coding-Enthusiast) [reply _data/devdocs/en/references/p2p_networking.md]: Added

File: _data/devdocs/en/references/p2p_networking.md [hunk 4]
```diff
@@ -1432,6 +1432,11 @@ The following service identifiers have been assigned.
...
+| 0x08  | NODE_WITNESS | This is a full node that can be asked for blocks and transactions including witness data.
```
[the remark this answers is not in the document]
Comment (Coding-Enthusiast) [reply _data/devdocs/en/references/p2p_networking.md]: Added

File: _data/devdocs/en/references/p2p_networking.md (hunk 1 above)
Comment (Coding-Enthusiast) [reply _data/devdocs/en/references/p2p_networking.md]: Is there any decent link to be added for this one?

Comment (harding) [reply _data/devdocs/en/references/p2p_networking.md]: I guess this, but it seems quite underspecified to me: https://github.com/BitcoinUnlimited/BUIP/blob/master/010.mediawiki

**Comment by wbnns** [comment]
This is scheduled to be merged on Friday, March 15th.

**Pull request merged**